.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = pfhub/_version.py
```

Comment:
The "⛔" at the end is because the file ends at the end of a line, not on a new/blank line, in conflict with the POSIX standard.